scrapy/link.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, url, text='', fragment='', nofollow=False):
         self.text = text
         self.fragment = fragment
         self.nofollow = nofollow
+        self.encoding = encoding
```

Comment:
so your comment is on https://github.com/scrapy/scrapy/blob/link-encoding/scrapy/linkextractors/lxmlhtml.py#L60, right?